src/app/components/shared/footer/footer.component.html

```diff
@@ -1,73 +1,74 @@
 <div class="footer">
-<div class="container">
-    <div
-      class="footer-blocks flex flex-wrap justify-content-between xl:justify-content-between lg:justify-content-around md:justify-content-around sm:justify-content-around"
-    >
+<div class="text-white py-5 mt-8">
```

Comment:
for class mt-8 it will be margin-top = 80px but in class="footer" you use margin-top = 8px

Reply:
Fixed